src/main/java/org/gradle/profiler/mutations/ClearGradleUserHomeMutator.java

```diff
@@ -3,35 +3,17 @@
 import org.gradle.profiler.BuildMutator;
 
 import java.io.File;
-import java.io.IOException;
-import java.io.UncheckedIOException;
-import java.nio.file.Files;
+import java.util.List;
 
-public class ClearGradleUserHomeMutator extends AbstractScheduledMutator {
-    private final File gradleUserHome;
+import static java.util.Collections.singletonList;
 
-    public ClearGradleUserHomeMutator(File gradleUserHome, Schedule schedule) {
-        super(schedule);
-        this.gradleUserHome = gradleUserHome;
-    }
+public class ClearGradleUserHomeMutator extends ClearDirectoryMutator {
 
-    @Override
-    protected void executeOnSchedule() {
-        System.out.printf("> Cleaning Gradle user home: %s%n", gradleUserHome.getAbsolutePath());
-        if (!gradleUserHome.exists()) {
-            throw new IllegalArgumentException(String.format(
-                "Cannot delete Gradle user home directory (%s) since it does not exist",
```

Comment:
💭  Since the clear-dir mutator is lenient in the case of the missing directory, the GUH clearing mutator is now also lenient, while it previously wasn't. But I don't see a big reason, why it should blow up. The GUH can be created by the first warm-up run, if required. It may not exist before-hand, and it should be fine.